packages/sdk/src/models/swaps.ts

```diff
@@ -117,6 +140,110 @@ export default class Swap {
     });
   };
 
+
+  /** Three substrate join_pool_xxx functions in one
+  */
+  joinPoolMultifunc = async (
```

Comment:
This one will need a lot of documentation, I'm not entirely sure we should have this instead of 3 separate functions which are clearly documented when each one should be used.

Reply:
Let's do both - safety plugs and clearly labelled extension cords